components/clarity-repl/src/repl/session.rs

```diff
@@ -823,14 +856,21 @@ impl Session {
             }
         };
 
-        let new_height = self.advance_chain_tip(count);
+        let new_height = self.advance_burn_chaintip(count);
         format!("{} blocks simulated, new height: {}", count, new_height)
             .green()
             .to_string()
     }
 
+    pub fn advance_stacks_chaintip(&mut self, count: u32) -> Result<u32, String> {
+        self.interpreter.advance_stacks_chaintip(count)
+    }
+    pub fn advance_burn_chaintip(&mut self, count: u32) -> u32 {
+        self.interpreter.advance_burn_chaintip(count)
+    }
+    // Not exposed/documented but still supported for convenience
```

Comment:
Not sure this comment is needed.
It's "exposed" as a pub function and used internally.